Week1/homework/index.html

```diff
@@ -1 +1,17 @@
-<!-- replace this with your HTML content -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Javascript2 HomeWork-Week1 </title>
+</head>
+
+<body>
+
```

Comment:
I'm missing the information from the book, i.e. author, title, and language.  It seems the cover is covering up these fields.